pkg/agent/openflow/client.go

```diff
@@ -408,20 +416,24 @@ func (c *client) UninstallServiceGroup(groupID binding.GroupIDType) error {
 	return nil
 }
 
-func (c *client) InstallEndpointFlows(protocol binding.Protocol, endpoints []proxy.Endpoint, isIPv6 bool) error {
+func generateEndpointFlowCacheKey(endpointIP string, endpointPort int, protocol binding.Protocol) string {
+	return fmt.Sprintf("Endpoints_%s_%d_%s", endpointIP, endpointPort, protocol)
+}
+
+func generateServicePortFlowCacheKey(svcIP net.IP, svcPort uint16, protocol binding.Protocol) string {
+	return fmt.Sprintf("Service_%s_%d_%s", svcIP, svcPort, protocol)
+}
+
+func (c *client) InstallEndpointFlows(protocol binding.Protocol, endpoints []proxy.Endpoint) error {
```

Comment:
I removed the isIPv6 argument, as I could not figure out why it is needed. Let me know if I am wrong. @weiqiangt